src/main/java/uk/gov/hmcts/reform/blobrouter/data/model/Event.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.blobrouter.data.model;
+
+public enum Event {
+    FILE_RECEIVED,
```

Comment:
Not sure we can catch `FILE_RECEIVED` event. We'll either have DISPATCHED, REJECTED, RECEIVED_DUPLICATE or FAILED_TO_PROCESS as the first event.

Reply:
perhaps rename? what i meant was smth like `FILE_START_PROCESSING`. and can be inserted once processing checker has allowed to continue - don't want to end up with godzillion entries

Reply:
Renaming sounds good